daprdocs/content/en/developing-applications/building-blocks/state-management/howto-state-query-api.md

```diff
@@ -0,0 +1,140 @@
+---
+type: docs
+title: "How-To: Query the state"
+linkTitle: "How-To: Query the state"
+weight: 250
+description: "API for querying state stores"
+---
+
+{{% alert title="alpha" color="warning" %}}
+The state query API is in **alpha** stage.
+{{% /alert %}}
+
+## Introduction
+
+The state query API provides a way of querying the key/value data stored in state store components. This query API is not a replacement for a complete query language, and is focused on retrieving, filtering and sorting key/value data that you have saved through the state management APIs.
+
+This query API does not support querying of actor state stored in a state store. For that you need to use the query API for the specific database.
+See [querying actor state]({{< ref "state-management-overview.md#querying-actor-state" >}}).
+
+You can find additional information in the [resources]({{< ref "#resources" >}}) section.
+
+## Querying the state
+
+You submit query requests via HTTP POST/PUT or gRPC.
+The body of the request is the JSON map with 3 entries: `filter`, `sort`, and `pagination`.
+
+The `filter` section specifies the query conditions in the form of a tree of key/value operations, where the key is the operator and the value is the operands.
+
+The following operations are supported:
+| Operator | Operands | Description |
+|----------|-----------|-------------|
+| `EQ`       | key:value  | key == value|
+| `IN`        | key:[]value| key == value[0] OR key == value[1] OR ... OR key == value[n] |
+| `AND`    | []operation|  operation[0] AND operation[1] AND ... AND operation[n] |
+| `OR`      | []operation|  operation[0] OR operation[1] OR ... OR operation[n] |
+
+The `sort` is an optional section is an ordered array of `key:order` pairs, where `key` is a key in the state store, and the `order` is an optional string indicating sorting order: `"ASC"` for ascending and `"DESC"` for descending. If omitted, ascending order is the default.
+
+The `pagination` is an optional section containing `limit` and `token` parameters. `limit` sets the page size. `token` is an iteration token returned by the component, and used in consecutive queries.
+
+Below is an example of the query request JSON body.
+
+This query retrieves all records where people either (1) belong to "Dev Ops" organization, or (2) belong to "Finance" and reside in Washington or California.
```

Comment:
I still think this description is not clear enough. For example the field key name is `person.org` where the value is "Dev Ops". I just want to be clear that `person.org` is different from just saying people.

Add that the results are returned sorted first in descending alphabetically order by `state` and then by ascending alphabetically order by `person.id` value